client_plugin/Makefile

```diff
@@ -80,6 +81,12 @@ PLUGIN := github.com/$(GOPATH_ORG)/$(GOPATH_PLUGNAME)
 # test location, picking up E2E tests
 E2E_Tests := github.com/$(GOPATH_ORG)/$(PLUGNAME)/tests/e2e
 
+# Pre upgrade test location
```

Comment:
L84-L88 is not needed .. as you have added correct tag to identify tests ... it can be control during run time.